functorch/_src/eager_transforms.py

```diff
@@ -243,12 +243,16 @@ def vjp(func: Callable, *primals, has_aux=False):
             primals_out = func(*diff_primals)
 
             if has_aux:
+                # TODO: Add a test for that once vjp/jvp both have has_aux option
```

Comment:
Why does the test have to wait until later?

Reply:
I think I thought about jacrev/jacfwd as they are tested with a decorator. Yes, we can add vjp test here as well